app/src/main/res/layout/fragment_info.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+xmlns:tools="http://schemas.android.com/tools"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+tools:context="com.mobileproto.hireddit.hireddit.visuals.InfoFragment"
+android:background="@color/colorBackground"
+android:paddingTop="15dp"
+android:paddingLeft="15dp"
+android:paddingRight="15dp"
+android:paddingBottom="50dp">
+
```

Comment:
idk as an android user i always use the built in back button, but i guess for someone who's new to android it might be good. i don't feel strongly about this either way